test/cypress/integration/request-to-go-live/index_spec.js

```diff
@@ -60,14 +60,15 @@ describe('Request to go live: index', () => {
       cy.visit(requestToGoLivePageUrl)
 
       cy.get('h1').should('contain', 'Request to go live')
+      cy.get('h1 + p').should('contain', 'Complete these steps to start taking real payments')
 
-      cy.get('#request-to-go-live-step-1 > h3').should('exist')
```

Comment:
In the original code it is not obvious here why we assert on existence and not on content, the idea was not to depend on text/content of each step in case there are minor content changes / or major which should not break the test. That is why I prefer to rely on ids not on content.
But, after discussion with @alexbishop1 I understand that it is not clear why we have this assertions and which step is what.
We discussed to change the ids to be more descriptive or to leave the content as it is highly coupled with implementation (as steps).

Thanks

Reply:
Just pushed a commit that asserts less on text and makes the IDs more descriptive